src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newObject = {};
+
+  for (const key in robot) {
+    if (newObject.hasOwnProperty(robot[key])) {
+      return null;
+    } else {
+      newObject[robot[key]] = key;
+    }
```

Comment:
`else` is redundant here. The function will stop if the first condition will be true
```suggestion
    }
    
    newObject[robot[key]] = key;
```

Reply:
Understand, thank you!